allennlp_models/structured_prediction/dataset_readers/universal_dependencies.py

```diff
@@ -111,7 +111,7 @@ def text_to_instance(
                 [x[0] for x in dependencies], text_field, label_namespace="head_tags"
             )
             fields["head_indices"] = SequenceLabelField(
-                [x[1] for x in dependencies], text_field, label_namespace="head_index_tags"
+                [x[1] for x in dependencies], text_field, label_namespace="head_indeices"
```

Comment:
I dug into this trying to figure out what's going on here, and I'm quite confident that this label namespace is entirely ignored, because the input to the field is a list of integers.  The vocab namespace that's referenced here is never read in the model, and the field itself is never used as if it's a string.  It could alternatively have been a `ListField[IndexField]`, and that would be more accurate for what's actually going on here.

```suggestion
                [x[1] for x in dependencies], text_field, label_namespace="ignored"
```

Reply:
After reading your reply, I checked this again and realized that this was my mistake. I was actually using a version of this code with a slight modification (which was the actual cause of the issue) and jumped to a wrong conclusion. I'm sorry for the trouble. I'd like it being `head_index_tags` rather than `ignored`, so I reverted the code.